### PR TITLE
Added algorithms in cp.md

### PR DESCRIPTION
**Pull request by sumitkr2000**
Hey @onecompiler-ops please review my pull request and check if it is corrected now. I have removed the advanced topics and added only relevent ones. I tried to keep it simple and concise. Hoping that you will approve my change. Thanks!

### CODE REVIEW

File: cpp.md [hunk 1]
```diff
@@ -274,8 +278,57 @@ typedef data-type name;
 typedef unsigned int distance;  // typedef of int 
 ```
 
-## Vectors
-Vectors are same as dynamic arrays. They will be resized on element inseertions & deletions.
+## Sort one-line
```
Comment (onecompiler-ops) [inline cpp.md:281]: This is a very specific use-case, does not belong in Cheat Sheets, please wite this as a post instead

File: cpp.md [hunk 2]
```diff
@@ -274,8 +278,57 @@ typedef data-type name;
...
+|#include<stdio.h>   | It is used to perform input and output operations|
+|#include<string.h>  | It is used to perform various string operations|
+|#include<math.h>    | It is used to perform mathematical operations|
+|#include<iomanip.h> | It is used to access set() and setprecision()|
+|#include<signal.h>  | It is used to perform signal handling functions like sign|
+|#include<stdarg.h>  | It is used to perform standard argument functions|
+|#include<errno.h>   | It is used to perform error handling operations like errno|
+|#include<fstream.h> | It is used to control the data to read from a file|
+|#include<time.h>    | It is used to perform functions related to date() and time|
+<<<<<<< HEAD
```
Comment (onecompiler-ops) [inline cpp.md:322]: Fix merge conflicts

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: cpp.md (hunk 2 above)
Comment (sumitkr2000) [reply cpp.md:322]: Ok.

File: cpp.md (hunk 1 above)
Comment (sumitkr2000) [reply cpp.md:281]: Hey @onecompiler-ops Please reply my question ! I want to contribute to this repo.

**Comment by sumitkr2000** [comment]
Hey, @onecompiler-ops I hope to get a reply on this issue so that i can work on it. Thank you!

**Comment by onecompiler-ops** [comment]
Please fix the comments and merge conflicts

**Comment by sumitkr2000** [comment]
@onecompiler-ops I have solved the merge conflicts, kindly review the changes and tell me if there is any more changes to be made? Otherwise please merge my pull request.

File: cpp.md [hunk 3]
```diff
@@ -380,47 +438,26 @@ int main() {
 }
 
 ```
-## Sort one-line
```
[the remark this answers is not in the document]
Comment (sumitkr2000) [reply cpp.md:383]: @onecompiler-ops I have only changed the place of sort-one line evrything is looking in preview in the cpp.md file. You can see the below screenshot for a better look:-

Screenshot (343)

Screenshot (344)


Let me know if there is any issue. Thank you!

[the remark this answers is not in the document]
Comment (sumitkr2000) [reply cpp.md:383]: @onecompiler-ops ??

**Comment by onecompiler-ops** [comment]
@sumitkr2000  There are many merge conflicts, resolve them or create a new clean PR

**Comment by sumitkr2000** [comment]
@onecompiler-ops Okk this time I will add only algorithm section. I will not touch any other section.

**Pull request closed without merging**